packages/react/__tests__/src/components/Toast/index.js

```diff
@@ -246,3 +246,16 @@ test('dismiss control is not rendered when dismissible is `false`', (done) => {
     done();
   }); // wait for animation timeouts / async setState calls
 });
+
+test('allows for custom class names', () => {
+  const mountElement = document.createElement('div');
+  document.body.appendChild(mountElement);
+  const wrapper = mount(
+    <Toast {...defaultProps} show={true} className="custom-class" type="info">
+      {'hi'}
+    </Toast>,
+    { attachTo: mountElement }
+  );
+
+  expect(wrapper.find('.Toast').hasClass('custom-class')).toBeTruthy();
```

Comment:
I know this is probably more than what you bargained for, but we've been [migrating to use RTL](https://github.com/dequelabs/cauldron/blob/develop/CONTRIBUTING.md#unit-tests)

Additionally, this test should probably cover that the extra class is added and keeps the existing class.

Reply:
No worries, more comfortable with RTL these days than enzyme to be honest. Is the expectation that the entire test suite be rewritten in RTL, or just new test cases (with others converted over time)? 

I don't have strong feelings either way, just want to clarify what's needed.